docs/development.md

```diff
@@ -1,128 +1,119 @@
-# Garden React Development
+# Garden Development
 
-This is a multi-package repo which uses [Lerna](https://lernajs.io/) to
-manage shared and cross-package dependencies.
+You're here because you want to get into the codebase. That's great. This
+guide will help you dig into the sensational Garden dirt. If you haven't
```

Comment:
> into the sensational Garden dirt

Could we drop the excess descriptives here? This is a development doc.

Reply:
Y'all are no fun 😜